jumanji/environments/packing/bin_pack/conftest.py

```diff
@@ -76,6 +78,106 @@ def __call__(self, key: chex.PRNGKey) -> State:
             sorted_ems_indexes=jnp.arange(self.max_num_ems, dtype=jnp.int32),
             # For deterministic instance generators we always set the key to 0.
             key=jax.random.PRNGKey(0),
+            nb_items=3,
+        )
+
+
+class DummyConstrainedGenerator(DummyGenerator):
+    """Dummy instance generator used for testing. It outputs a constant instance with a 20-ft
+    container and 3 items: two identical items and a different third one to be able to
+    test item aggregation.
+    """
+
+    def __init__(self) -> None:
+        """Instantiate a dummy `Generator` with 3 items and 10 EMSs maximum."""
+        super(DummyGenerator, self).__init__(
+            max_num_items=3, max_num_ems=10, container_dims=TWENTY_FOOT_DIMS
+        )
+
+    def __call__(self, key: chex.PRNGKey) -> State:
+        """Returns a fixed instance with 3 items, 10 EMSs and a 20-ft container.
+
+        Args:
+            key: random key not used here but kept for consistency with parent signature.
+
+        Returns:
+            State.
+        """
+        del key
+        container = make_container(TWENTY_FOOT_DIMS)
+        return State(
+            container=container,
+            ems=jax.tree_util.tree_map(
+                lambda x: jnp.array([x] + (self.max_num_ems - 1) * [0], jnp.int32),
+                container,
+            ),
+            ems_mask=jnp.array([True] + (self.max_num_ems - 1) * [False], bool),
+            items=Item(
+                # The 1st and 2nd items have the same shape.
```

Comment:
Yes this is still the case, both items have the dimensions (1000, 700, 900). It's in 2D because I didn't want to modify the whole logic and I opted for flattening the observations only when I needed them to be flat .i.e only inside the actor_critic.py file.